alacritty_terminal/src/renderer/mod.rs

```diff
@@ -994,12 +994,21 @@ impl<'a> RenderApi<'a> {
 
     pub fn render_cell(&mut self, cell: RenderableCell, glyph_cache: &mut GlyphCache) {
         let chars = match cell.inner {
-            RenderableCellContent::Cursor((cursor_style, ref raw)) => {
+            RenderableCellContent::Cursor(cursor_key) => {
                 // Raw cell pixel buffers like cursors don't need to go through font lookup
-                let glyph = glyph_cache
-                    .cursor_cache
-                    .entry(cursor_style)
-                    .or_insert_with(|| self.load_glyph(raw));
+                let metrics = &glyph_cache.metrics;
```

Comment:
Instead of borrowing and dereferencing it right after, we probably should just use `glyph_cache.metrics` when calling `get_cursor_glyph`.

If this is done because of the borrow checker, we should at least remove the borrow so we don't borrow and then dereference immediately after.

Reply:
The borrow checker won't let us just write `glyph_cache.metrics`, but the borrow was unnecessary